templates/Dockerfile-template.nvidia

```diff
@@ -6,19 +6,19 @@
 #
 #
 
-FROM    nvidia/cudagl:9.2-devel-ubuntu18.04 AS devel-base
+FROM    nvidia/cuda:10.2-devel-ubuntu18.04 AS devel-base
 
-ENV	    NVIDIA_DRIVER_CAPABILITIES compat32,compute,video
+ENV	    NVIDIA_DRIVER_CAPABILITIES compute,utility,video
 WORKDIR     /tmp/workdir
 
 RUN     apt-get -yqq update && \
         apt-get install -yq --no-install-recommends ca-certificates expat libgomp1 && \
         apt-get autoremove -y && \
         apt-get clean -y
 
-FROM        nvidia/cudagl:9.2-runtime-ubuntu18.04 AS runtime-base
+FROM        nvidia/cuda:10.2-devel-ubuntu18.04 AS runtime-base
```

Comment:
use runtime on the final image ?

Reply:
Of course, somehow I missed this!

Reply:
Fixed

Reply:
still there : https://github.com/Jalle19/ffmpeg/blob/nvidia-updates/templates/Dockerfile-template.nvidia#L19